src/ui/public/courier/__tests__/saved_object.js

```diff
@@ -82,19 +83,108 @@ describe('Saved Object', function () {
     const savedObject = new SavedObject(config);
     return savedObject.init();
   }
-
-  beforeEach(ngMock.module('kibana'));
-  beforeEach(ngMock.inject(function (es, Private) {
+  beforeEach(ngMock.module('kibana',
+
+    // The default implementation of safeConfirm uses $timeout which will cause
+    // the test environment to hang.
+    function ($provide) {
+      $provide.decorator('safeConfirm', () => {
+        return (message) => {
```

Comment:
Interesting, I didn't know you could do that.  But does that fit here?  I'm returning a function, not an object literal.